### DISCUSSION TITLE
Daisy UI color change

### DISCUSSION THREAD

**Discussion by Bamschool** [fuma-nama/fumadocs#622] [Q&A]
Hi! When I tried to apply Daisy UI, the colors in the docs changed. Is there any way to fix this? Thanks!

```
const { createPreset } = require("fumadocs-ui/tailwind-plugin");

/** @type {import('tailwindcss').Config} */
module.exports = {
  content: [
    "./components/**/*.{ts,tsx}",
    "./app/**/*.{ts,tsx}",
    "./content/**/*.{md,mdx}",
    "./mdx-components.{ts,tsx}",
    "./node_modules/fumadocs-ui/dist/**/*.js",
  ],
  presets: [createPreset()],
  plugins: [require("daisyui")],

  // plugins: [require("daisyui")],
};
```

<img width="286" alt="스크린샷 2024-07-22 오후 8 48 35" src="https://github.com/user-attachments/assets/773d7305-3d5d-40fa-b45b-c7f79f89767b">

**Comment by fuma-nama** [comment]
Probably some color names conflicted with each other, we are still planning to add a prefix to all built-in color names. For now you have to override these color names back to Fumadocs UI default (you can copy the source code from https://github.com/fuma-nama/fumadocs/blob/dev/packages/ui/src/tailwind-plugin.ts)

**Reply by Bamschool** [reply]
thank you! it  works!!
